features/rni.feature

```diff
@@ -0,0 +1,202 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van de categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
+      | code | omschrijving                                      |
+      | 101  | Belastingdienst (inzake heffingen en toeslagen)   |
+      | 201  | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+
+
+    Scenario: RNI-deelnemer op geleverd gegeven
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000012                                   |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 000000012                         |
+      | fields              | naam.voornamen,naam.geslachtsnaam |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
```

Comment:
Deze 'Dan' step genereert netjes de response waartegen de actual response getest wordt, de daaropvolgende 'En' step heeft daar echter geen invloed op. Doordat ook de GBA API geen RNI gerelateerde properties verstrekt heeft dat tot gevolg dat het scenario bij mij (met bovenstaande wijzigingen) succesvol door de test komt. Eigenlijk wordt dit scenario dus niet correct verwerkt.

Reply:
Ik zie nu dat in 'fields' het veld 'rni' ontbreekt.

Reply:
Het woordje 'bug' had ik misschien niet mogen gebruiken, het is eerder een stukje functionaliteit die nog niet ingevuld is.

Reply:
rni support is nu toegevoegd aan automation code